playbooks/common/openshift-cluster/upgrades/post_control_plane.yml

```diff
@@ -70,4 +70,11 @@
     when:
     - __shared_resource_viewer_protected | default(false)
 
+# Tuned profile are removed when removing tuned-profiles-atomic-openshift-node rpm
```

Comment:
I think this would be better as an additional task at https://github.com/openshift/openshift-ansible/blob/6b4ecfac78d9a03139d903c867cc20ed1f69b182/playbooks/openshift-master/private/upgrade.yml#L347

rather than a new play.